tensorflow/core/kernels/scatter_functor.h

```diff
@@ -188,6 +189,7 @@ struct AssignSYCL<scatter_op::UpdateOp::MAX> {
 }  // namespace scatter_op
 
 namespace functor {
+#define kMaxLocks 1024
```

Comment:
Let's make this a constant variable.
```suggestion
const Index kMaxLocks 1024;
```
Then you won't have to cast `kMaxLocks` to `Index` in `std::min`.